content/getting-started/yaml.md

```diff
@@ -143,14 +143,16 @@ Note that `mac_pro`, `linux`, and `linux_x2` are only available for teams and us
 
 ### Environment
 
-`environment:` contains all the environment variables and enables to specify the version of Flutter, Xcode, CocoaPods, Node, npm and Ruby used for building. This is also where you can add credentials and API keys required for [code signing](../code-signing-yaml/signing). Make sure to [encrypt the values](../building/encrypting) of variables that hold sensitive data. 
+`environment:` contains all the environment variables and enables to specify the version of Flutter, Xcode, CocoaPods, Node, npm and Ruby used for building. This is also where you can add credentials and API keys required for [code signing](../code-signing-yaml/signing). See the explanation for [encrypting environment variables](../building/encrypting) and using [environment variable groups](../building/environment-variable-groups/). 
```

Comment:
I would personally redo this section with more depth, I think it is quite bad as it is.. I would consider removing some of the example and dividing into sections, at least in some form:

Defining individual environment variables

```yaml
environment:
  vars: # Define your environment variables here
    PUBLIC_ENV_VAR: 'value here'
    SECRET_ENV_VAR: Encrypted(...)
```

Defining environment variable groups

```yaml
environment:
  groups: # Import UI defined environment variable groups here
    - production
```

Defining environment versions

```yaml
environment:
  flutter: stable # Define the channel name or version (e.g. v1.13.4)
  xcode: latest # Define latest, edge or version (e.g. 11.2)
  cocoapods: 1.9.1 # Define default or version
  node: 12.14.0 # Define default, latest, current, lts, carbon (or another stream), nightly or version
  npm: 6.13.7 # Define default, latest, next, lts or version
  ndk: r21d # Define default or revision (e.g. r19c)
  java: 1.8 # Define default, or platform version (e.g. 11)
  ruby: 2.7.2 # Define default or version
```

Using different environment sections in conjunction

```yaml
environment:
  vars: # Define your environment variables here
    PUBLIC_ENV_VAR: 'value here'
  groups: # Import UI defined environment variable groups here
    - staging
  xcode: latest # Define latest, edge or version (e.g. 11.2)
```

Reply:
Great suggestion, I went ahead and did this.